lib/jxl/base/arch_macros.h

```diff
@@ -28,6 +28,12 @@ namespace jxl {
 #define JXL_ARCH_ARM 0
 #endif
 
+#if defined(__riscv) && __riscv_xlen == 64
+#define JXL_ARCH_RISCV64 1
```

Comment:
Where this macro will be used?

Reply:
I was working on Chromium build and got some errors when compiling libjxl, this changes will help Chromium compilation but we have yet to test the browser functionality with libjxl, we might need this in future enhancement for Chromium browser.

Reply:
I am also interested in the build error, since libjxl compiles just fine on riscv64/Debian/sid. Thanks

Reply:
Technically I see that `JXL_ARCH_PPC` is not used anywhere...

Reply:
Thanks for you note, I have rebased the Chromium source code and retried. libjxl is compiling fine there. Thanks. I will close this PR.